exe/govuk_security_audit

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
```

Comment:
It's to separate gem-supplied exe files from bundler-generated binstubs http://bundler.io/blog/2015/03/20/moving-bins-to-exe.html